kde/kdesvn/files/patch-fix-build.diff

```diff
@@ -0,0 +1,21 @@
+--- CMakeLists.txt	2010-03-21 00:08:10.000000000 +0800
++++ CMakeLists.txt	2024-07-18 23:33:39.000000000 +0800
+@@ -27,6 +27,8 @@
+ ENDIF (CMAKE_MAJOR_VERSION EQUAL 2 AND CMAKE_MINOR_VERSION EQUAL 4)
+ SET(BASE_VERSION 1.6.0)
+ 
++cmake_policy(SET CMP0004 OLD)
```

Comment:
I have no idea what this does?

Reply:
It is pointed out in the ticket which I referred above.
Here: https://trac.macports.org/ticket/70158#comment:3

Reply:
Could you patch out the whitespace instead or is this the cleanest way to deal with it?

Reply:
This is an easy and minimal fix, which works fine. I have no idea what CMake is unhappy about in particular and where those spaces come from. Given that this port uses Qt4, it is appropriate IMO to use older policies.